build.sbt

```
@@ -15,12 +15,14 @@ lazy val commonSettings = Seq(
 )
 
 val scorexVersion = "e1142a21-SNAPSHOT"
-val sigmaStateVersion = "v2.1-b5a50364-SNAPSHOT"
-
+val sigmaStateVersion = "soft-forkability2-1a0e5d54-SNAPSHOT"
+val ergoWalletVersion = "soft-forkability2-1be4a528-SNAPSHOT"
 // for testing current sigmastate build (see sigmastate-ergo-it jenkins job)
 val effectiveSigmaStateVersion = Option(System.getenv().get("SIGMASTATE_VERSION")).getOrElse(sigmaStateVersion)
 
 libraryDependencies ++= Seq(
+  ("org.ergoplatform" %% "ergo-wallet" % ergoWalletVersion)
+    .exclude("ch.qos.logback", "logback-classic"),
   ("org.scorexfoundation" %% "sigma-state" % effectiveSigmaStateVersion)
```

```suggestion
  ("org.scorexfoundation" %% "sigma-state" % effectiveSigmaStateVersion).force()
```
Otherwise in sigma's ergo tests injected version (via `SIGMASTATE_VERSION` env var) "conflicts" with sigmastate version from `ergo-wallet` (indirect dep) and gets ignored randomly.

---

Fixed in https://github.com/ergoplatform/ergo/pull/741/commits/de4e2a1e568bbd8f77745b0030340e816fd74666